src/objective-c/tests/BUILD

```diff
@@ -101,6 +101,7 @@ grpc_objc_testing_library(
     hdrs = ["InteropTests/InteropTests.h"],
     deps = [
         ":InteropTestsBlockCallbacks-lib",
+        ":TestBase-lib",
```

Comment:
one NIT: we may also add the dep to https://github.com/grpc/grpc/blob/48749f739e7dd91cb69c988d57a89ffe93d8b501/src/objective-c/grpc_objc_internal_library.bzl#L112  (similar to :RemoteTest target) as this seems to be a common base dep for all test tagets. But this can be in a follow up ;  )

Reply:
Let's do as a followup.